ir/inst_aggregate.go

```diff
@@ -90,6 +92,8 @@ type InstInsertValue struct {
 	Metadata
 }
 
+func (inst *InstInsertValue) Operands() []value.Value { return []value.Value{inst.X, inst.Elem} }
```

Comment:
Use `go fmt` to format the source file.

Reply:
I think this is valid under `go fmt`

Reply:
Because after I run `go fmt`, it still keep this result